.azure-pipelines/cuda-unit-test.yml

```diff
@@ -15,6 +15,9 @@ steps:
   - script: |
       python3 -m pip install .[test,torch]
     displayName: Install dependencies
+  - script: |
+      make cppbuild
+    displayName: Build benchmarks
```

Comment:
should we build these in unit test? there's no unit test for cpp code, and build process will be checked in docker build ci.

May be build and run these after there's integration tests?

Reply:
The cppbuild inside cuda-unit-test.yml is for python benchmarks, we should check something like 'if binary exists' and 'the output format'.